components/public-api-server/pkg/apiv1/identityprovider.go

```diff
@@ -9,6 +9,7 @@ import (
 	"fmt"
 
 	connect "github.com/bufbuild/connect-go"
+	"github.com/filiptronicek/subber"
```

Comment:
Do we really need a dependency to a personal project here?

I'm not convinced the extra dependency is really bringing any value here.

Reply:
I'm fine with copying the code over, let's do it.

Reply:
Given it's just a single function really, let's include it here. Also helps prevent a `leftpad` scenario

Reply:
@easyCZ are we ok with having the code in this file or do we move it somewhere else?